google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/spi/v1/PollingSubscriberConnection.java

```diff
@@ -22,8 +22,8 @@
 import com.google.api.stats.Distribution;
 import com.google.auth.Credentials;
 import com.google.cloud.Clock;
-import com.google.cloud.pubsub.spi.v1.MessagesProcessor.AcksProcessor;
-import com.google.cloud.pubsub.spi.v1.MessagesProcessor.PendingModifyAckDeadline;
+import com.google.cloud.pubsub.spi.v1.MessageDispatcher.AcksProcessor;
```

Comment:
I didn't notice before, but `AcksProcessor` should be `AckProcessor` to conform to English compound rules. I'll leave it to you if you want to do it in this PR or a separate one.